src/python2/sprettball/sprettball.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Sprettball
+level: 3
+Author: Ole Kristian Pedersen, Kodeklubben Trondheim
+---
+
+# Introduksjon {.intro}
+
+I denne oppgaven skal du lage en ballanimasjon, ved hjelp av det du har lært i
+[oppgaven om enkle objekter](../enkle_objekter.html). Dermed kan det være lurt
+å gå tilbake for å repetere, dersom du ikke husker hva du lærte der. I tillegg
```

Comment:
> Dermed kan det være lurt
+å gå tilbake for å repetere, dersom du ikke husker hva du lærte der.

Litt mye bruk av "lærte". Litt pirk, men "hva du lærte" er vagt. Altså, hva betyr det å lære?

Kan omformuleres til noe som dette:
Dersom du ikke husker objekter, kan du [gå tilbake](..link) og raskt repetere.